_format/1.1/index.md

```diff
@@ -360,10 +372,21 @@ response that includes the resource as the primary data.
 A "resource identifier object" is an object that identifies an individual
 resource.
 
-A "resource identifier object" **MUST** contain `type` and `id` members.
+A "resource identifier object" **MUST** contain a `type` member. A resource
+identifier object **MUST** also contain an `id` member, except when it contains
+a `lid` that identifies it as matching a new resource to be created on the
```

Comment:
Should this then state explicitly that `lid`s can only exist in request documents, not in response documents?

Reply:
`lid` can exist both in the request and the response object (that's sort of their purpose)

Reply:
Right, to communicate to the client what the server-generated `id` for a client-specified `lid` is. Of course. Silly me!